<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>931</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="main">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Played Note</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>480</y>
       <width>161</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_button">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>480</y>
       <width>151</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>TV Command</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>111</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>109</y>
       <width>801</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_1">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>70</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>On/Off</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>140</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>V+</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>220</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>V-</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>300</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ch+</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>380</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ch-</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_11">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>70</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 0</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_12">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>140</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_13">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>220</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_14">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>300</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 3</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_15">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>380</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>380</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 4</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_16">
     <property name="geometry">
      <rect>
       <x>477</x>
       <y>70</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>70</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 5</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_17">
     <property name="geometry">
      <rect>
       <x>477</x>
       <y>140</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>140</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 6</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_18">
     <property name="geometry">
      <rect>
       <x>477</x>
       <y>220</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>220</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 7</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_19">
     <property name="geometry">
      <rect>
       <x>477</x>
       <y>300</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>300</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 8</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_20">
     <property name="geometry">
      <rect>
       <x>477</x>
       <y>380</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Key 9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_31">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>70</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>70</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_32">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>140</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>140</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_33">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>220</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>220</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_34">
     <property name="geometry">
      <rect>
       <x>697</x>
       <y>300</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>300</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_35">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>380</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>803</x>
       <y>380</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_36">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>460</y>
       <width>97</width>
       <height>38</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>803</x>
       <y>460</y>
       <width>64</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
